alaoweb/2020.alaoweb.org
# support multiple award winners

* closes #272 
* changes winner data element to winners: [] array
* updates awards.html to expect this array
* as before introduces MORE spurious award winners

## to test: 
* set `showAwardWinners: true` in `config.yml`
* see handling of multiple awardees on the TEDDY award
